doc/neps/nep-0052-python-api-cleanup.rst

```diff
@@ -181,9 +230,20 @@ Details TBD, something like::
     numpy.doc
     numpy.matlib
     numpy.version
-    
+
     # To clean out or somehow deal with: everything in `numpy.lib`
 
+.. S: Are you thinking that even math.* will disappear out of the name
+   mainspace? That's quite a big change.  I like the principle you
+   proposed on Sebastian's PR above: one function, one home.
```

Comment:
No I wasn't thinking about that - also these functions are not present elsewhere? E.g. `np.math.acos` is not the same as `np.arccos`. `np.math` is a bit odd though, I don't think I've ever seen it used.

Reply:
Hrm, so maybe they warrant hiding a bit deeper.

Reply:
Yes, I'm not sure what the point is of those functions, will add a comment.